### PR TITLE
Add missing eval_metrics to hp validation.

### PR DESCRIPTION
**Pull request by ericangelokim**
*Issue #, if available:*

*Test*
```tox```
Adding container tests in pipelines

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/sagemaker_xgboost_container/algorithm_mode/metrics.py [hunk 1]
```diff
@@ -12,43 +12,15 @@
...
-                 regex=".*\\[[0-9]+\\].*#011validation-mse:(\\S+)"),
-        m.Metric(name="validation:ndcg",
-                 direction=m.Metric.MAXIMIZE,
-                 regex=".*\\[[0-9]+\\].*#011validation-ndcg:(\\S+)"),
-        m.Metric(name="validation:rmse",
-                 direction=m.Metric.MINIMIZE,
-                 regex=".*\\[[0-9]+\\].*#011validation-rmse:(\\S+)"))
+
+    metrics = [m.Metric(name="validation:{}".format(metric_name),
+                        direction=m.Metric.MAXIMIZE,
```
Comment (iyerr3) [inline src/sagemaker_xgboost_container/algorithm_mode/metrics.py]: Not clear to me on how direction is being set based on metric name

Comment (iyerr3) [reply src/sagemaker_xgboost_container/algorithm_mode/metrics.py]: We might have to split into two lists or have a tuple for each entry with second item indicating the direction.

Comment (ericangelokim) [reply src/sagemaker_xgboost_container/algorithm_mode/metrics.py]: I've added it back as it was before, with the addition of aucpr.

**Comment by iyerr3** [comment]
I liked your previous change to simplify the code. Was thinking we could split to `XGB_MAXIMIZE_METRICS` and `XGB_MINIMIZE_METRICS` to get the split behavior.

**Comment by ericangelokim** [comment]
I would prefer we have one source too. I think have those two lists makes sense.

**Pull request merged**